src/component/strategies/EditStrategy/EditStrategy.tsx

```diff
@@ -77,6 +77,7 @@ export const EditStrategy = () => {
             description="The strategy type and the parameters will be selectable when adding an activation strategy to a toggle in the environments.
             The parameter defines the type of activation strategy. E.g. you can create a type 'Teams' and add a parameter 'List'. Then it's easy to add team names to the activation strategy"
             documentationLink="https://docs.getunleash.io/advanced/custom_activation_strategy"
+            documentationLinkLabel="More about custom strategies"
```

Comment:
Same as above ☝🏼  Is "custom _activation_ strategies" better?

```suggestion
            documentationLinkLabel="More about custom activation strategies"
```